scripts/gulp/dev-server.js

```diff
@@ -3,19 +3,70 @@
 const fs = require('fs');
 const log = require('fancy-log');
 const urlParser = require('url');
+const Mustache = require('mustache');
 
 const { createServer, useSsl } = require('./create-server');
 
-function readmeText() {
-  return fs.readFileSync('./README.md', 'utf-8');
+const DOCUMENT_PATH = './test-documents/';
```

Comment:
I would tend to omit the trailing `/` here and instead require it to be added after references to `DOCUMENT_PATH` elsewhere, otherwise something like `${DOCUMENT_PATH}preformatted.txt` looks like it is adding `preformatted.txt` to the end of the filename rather than generating a reference to `preformatted.txt` in a particular directory.